napari/layers/base/base.py

```diff
@@ -2058,6 +2059,17 @@ def save(self, path: str, plugin: Optional[str] = None) -> List[str]:
 
         return save_layers(path, [self], plugin=plugin)
 
+    def __copy__(self):
+        """Create a copy of this layer.
+
+        Returns
+        -------
+        layer : napari.layers.Layer
+            Copy of this layer.
+        """
+        data, meta, layer_type = self.as_layer_data_tuple()
```

Comment:
I would then add a comment here saying that benchmarks will break if we change the copy behaviour.